### ISSUE TITLE
[FEATURE]: Display required keymap 

### ISSUE THREAD

**Issue by alifhaider** [webdevcody/code-racer#186]
### Feature Summary

It's currently too hard to notice what keys need to be pressed after pressing `Enter` for a new line. 

We could display a required keymap to pass the current snippet. This needs to have UI updates at every keypress so that I can know what the next required key is.

Like this

<img width="948" alt="Screenshot 2023-07-14 at 3 05 51 AM" src="https://github.com/webdevcody/code-racer/assets/33329896/3c6376eb-4dbc-4693-bf5b-be9e6f428ef0">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by webdevcody** [comment]
I'm not sure if this is still an issue, I think this might have been fixed.  Feel free to reopen if this is still an issue.

**Comment by Ragudos** [comment]
Yeah, I think this is fixed with the yellow background indicator.
